src/standalone/Vegetation/autotrophic_respiration.jl

```diff
@@ -0,0 +1,119 @@
+export AutotrophicRespirationParameters, AutotrophicRespirationModel
+
+abstract type AbstractAutotrophicRespirationModel{FT} <: AbstractCanopyComponent{FT} end
+
+"""
+    AutotrophicRespirationParameters{FT<:AbstractFloat}
+
+The required parameters for the autrophic respiration model.
+$(DocStringExtensions.FIELDS)
+"""
+struct AutotrophicRespirationParameters{FT <: AbstractFloat}
+    "Mean leaf nitrogen concentration (kg N (kg C)-1)"
+    nm::FT
+    "Carbon content of roots (?)" # not sure about units
+    Rc::FT
+    "Carbon content of stems (?)"
+    Sc::FT
+    "Specific leaf density (kg C m-2 [leaf])"
+    σl::FT
+    "Ratio root nitrogen to top leaf nitrogen (-), typical value 1.0"
+    μr::FT
+    "Ratio stem nitrogen to top leaf nitrogen (-), typical value 0.1"
+    μs::FT
+    "Nitrogen content of leafs" 
+    Nl::FT
+    "Nitrogen content of roots"
+    Nr::FT
+    "Nitrogen content of stems"
+    Ns::FT
+    "Factor to convert from mol CO2 to kg C" # Definitely not a parameter. not sure what to do.
+    f1::FT
+    "Factor of relative contribution or Rgrowth"
+    f2::FT 
+end
+
+"""
+    function AutotrophicRespirationParameters{FT}(;
+        nm = FT(),
+        Rc = FT(),
+        Sc = FT(),
+        σl = FT(),
+        μr = FT(1.0),
+        μs = FT(0.1),
+        Nl = FT(),
+        Nr = FT(),
+        Ns = FT(),
+        f1 = FT(), 
+        f2 = FT()        
+) where {FT}
+
+A constructor supplying default values for the AutotrophicRespirationParameters struct.
+"""
+function AutotrophicRespirationParameters{FT}(;
+    nm = FT(),
+    Rc = FT(),
+    Sc = FT(),
+    σl = FT(),
+    μr = FT(1.0),
+    μs = FT(0.1),
+    Nl = FT(),
+    Nr = FT(),
+    Ns = FT(),
+    f1 = FT(), 
+    f2 = FT()
+) where {FT}
+    return AutotrophicRespirationParameters{FT}(
+    nm,
+    Rc,
+    Sc,
+    σl,
+    μr,
+    μs,
+    Nl,
+    Nr,
+    Ns,
+    f1, 
+    f2
+    )
+end
+
+struct AutotrophicRespirationModel{FT} <: AbstractAutrophicRespirationModel{FT} # we could give it a more specific name...
+    parameters::AutotrophicRespirationParameters{FT}
+end
+
+ClimaLSM.name(model::AbstractAutotrophicRespirationModel) = :autotrophic_respiration
+ClimaLSM.auxiliary_vars(model::AutotrophicRespirationModel) = (:Ra) # Ra = Rd + Rpm + Rg
+ClimaLSM.auxiliary_types(model::AutotrophicRespirationModel{FT}) where {FT} = (FT,)
+
+"""
+
+"""
+function compute_autrophic_respiration(
+    model::AutotrophicRespirationModel,
+    LAI,
+    GPP,
+    Rd,
+    β,
+)
+
+    (;
+    nm,
+    Rc,
+    Sc,
+    σl,
+    μr,
+    μs,
+    Nl,
+    Nr,
+    Ns,
+    f1, 
+    f2) = model.parameters
+
+    Nl, Nr, Ns = nitrogen_content(nm, LAI, Rc, Sc, σ, μr, μs)
+    Rpm = plant_respiration_maintenance(Rd, β, Nl, Nr, Ns, f1) 
+    Rg = plant_respiration_growth(f2, GPP, Rpm)
+    Ra = Rd + Rpm + Rg # Should this be a function in canopy_parameterizations.jl or is it ok here?
```

Comment:
In this way we're double counting Rd. This should be instead:

Ra = Rpm + Rg

Reply:
Where are we double counting Rd? 
Is Rd equivalent to maintenance respiration or growth respiration?